Vagrantfile

```diff
@@ -875,3 +875,40 @@ Vagrant.configure("2") do |config|
     trigger.on_error = :continue
   end
 end
+
+class VVVScripts < Vagrant.plugin(2, :command)
```

Comment:
Other than the trigger files what would this be used for? The db backup and restore scripts should be in path, as should xdebug_on etc so `vagrant ssh -c "xdebug_on"` should be enough

Reply:
This can execute any script uploaded in the homebin, also custom ones.

I think that is a more simple alias compared to that command, but in case we can remove this code.